tests/abi/testabi_rel.c

```diff
@@ -95,6 +95,11 @@ static void test_keyCmp (void)
 	succeed_if (keyCmp (k1, k2) < 0, "find_me is smaller");
 	succeed_if (keyCmp (k2, k1) > 0, "find_me is smaller");
 
+	keySetName (k1, "user:user_a/b");
```

Comment:
I think this should fail.

Reply:
yes, this was indeed causing the trouble